packages/documentation-site/patternfly-docs/content/demos/Compass/CompassIntegrationsDemo.tsx

```diff
@@ -0,0 +1,825 @@
+        isCircle
+      >
+        <Flex
+          alignItems={{ default: "alignItemsCenter" }}
+          gap={{ default: "gapMd" }}
+        >
+          User Name
+          <Avatar src={imgAvatar} alt="" size="md" />
+        </Flex>
+      </MenuToggle>
+    )}
+  >
+    <DropdownList>{userDropdownItems}</DropdownList>
+  </Dropdown>
+  );
+
+  const headerContent = <CompassHeader 
+    logo={<a href="#" aria-label="Red Hat automations" tabIndex={0}><RHAutomationsLogo /></a>}
+    nav={navContent} 
+    profile={userDropdown}
+  />;
+
+  const cardViewContent = (
+    <>
+      <Toolbar>
+        <ToolbarContent>
+          <ToolbarGroup>
+            <ToolbarItem>
+              <SearchInput
+                aria-label="Integrations example search input"
+                placeholder="Filter by name"
+              />
+            </ToolbarItem>
+          </ToolbarGroup>
+          <ToggleGroup>
+            <ToggleGroupItem
+              icon={<ThIcon />}
+              aria-label="Grid view"
+              isSelected={activeDisplay === "card"}
+              onChange={() => {
+                setActiveDisplay("card");
+              }}
+            ></ToggleGroupItem>
+            <ToggleGroupItem
+              icon={<ListIcon />}
+              aria-label="Table view"
+              isSelected={activeDisplay === "table"}
+              onChange={() => {
+                setActiveDisplay("table");
+              }}
+            ></ToggleGroupItem>
+          </ToggleGroup>
+          <ToolbarItem variant="pagination" align={{ default: "alignEnd" }}>
+            <Pagination
+              itemCount={523}
+              perPage={20}
+              page={1}
+              onSetPage={() => { }}
+              widgetId="pagination-options-card-view"
+              onPerPageSelect={() => { }}
+              isCompact
+            />
+          </ToolbarItem>
+        </ToolbarContent>
+      </Toolbar>
+      <Gallery hasGutter>
+        {integrations.map((product) => (
+          <Card
+            isCompact
+            isFullHeight
+            key={product.name}
+            id={product.name.replace(/ /g, "-")}
+          >
+            <CardHeader
+              actions={{
+                actions: (
+                  <>
+                    <Dropdown
+                      isOpen={false}
+                      onOpenChange={() => { }}
+                      toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+                        <MenuToggle
+                          ref={toggleRef}
+                          aria-label={`${product.name} actions`}
+                          variant="plain"
+                          icon={<EllipsisVIcon />}
+                        />
+                      )}
+                      popperProps={{ position: "right" }}
+                    />
+                  </>
+                ),
+              }}
+            >
+              <Content component="h4">{product.name}</Content>
```

Comment:
```suggestion
              <Content component="h2">{product.name}</Content>
```

This will make the text a little bigger in the Cards, so idk if we'd want to use a Title instead here to contorl the font size and still use the h2 heading.